IdentityCore/src/webview/embeddedWebview/ui/mac/MSIDWebviewUIController.m

```diff
@@ -37,6 +39,14 @@ @interface MSIDWebviewUIController ( ) <NSWindowDelegate>
 
 @implementation MSIDWebviewUIController
 
++ (void)initialize
```

Comment:
to discuss: should we use a class variable here instead? We use it more commonly in other places

Reply:
it can be a little more complicated here to use class property in this case, 
if passed on webview is used, the class property webviewconfig will not be used and this can cause more confusion.
plus, one can always obtain config by accessing the wkwebview property. 

this will be a different story when and if we support custom webview configuration. 

I will create an issue to track this #251